services/permits/app/pipelines/permit_condition_search/permit_condition_search_pipeline.py

```diff
@@ -0,0 +1,207 @@
+search_azure_endpoint = Secret.from_env_var("AZURE_SEARCH_SERVICE_ENDPOINT", strict=True)
+
+ca_cert = os.environ.get("ELASTICSEARCH_CA_CERT", None)
+host = os.environ.get("ELASTICSEARCH_HOST", None) or "https://elasticsearch:9200"
+username = os.environ.get("ELASTICSEARCH_USERNAME", "")
+password = os.environ.get("ELASTICSEARCH_PASSWORD", "")
+
+
+assert api_key and api_key is not None
+assert deployment_name
+assert base_url
+assert api_version
+
+
+doc_metadata_fields = {
+    "category": str,
+    "issue_date": datetime,
+    "permit": str,
+    "mine_number": str,
+    "mine_name": str,
+    "document_name": str,
+    "document_manager_guid": str,
+    "step": str,
+    "step_path": str,
+}
+
+extra_field_config = {
+    "category": {"filterable": True, "sortable": True, "facetable": True},
+    "issue_date": {"filterable": True, "sortable": True, "facetable": True},
+    "permit": {"filterable": True, "sortable": True, "facetable": True},
+    "mine_number": {"filterable": True, "sortable": True, "facetable": True},
+    "mine_name": {"filterable": True, "sortable": True, "facetable": True},
+    "document_name": {"filterable": True, "sortable": True, "facetable": True},
+    "document_manager_guid": {"filterable": True, "sortable": True, "facetable": True},
+    "step": {"filterable": True, "sortable": True, "facetable": True},
+    "step_path": {"filterable": True, "sortable": True, "facetable": True},
+}
+
+vector_search_config = VectorSearch()
+
+
+def create_azure_search_document_store():
+    return AzureSearchDocumentStore(
+        extra_field_config=extra_field_config,
+        api_key=search_api_key,
+        azure_endpoint=search_azure_endpoint,
+        index_name="permit-conditions",
+        embedding_dimension=3072,
+        metadata_fields=doc_metadata_fields,
+        vector_search_configuration=vector_search_config,
+    )
+
+def create_elasticsearch_document_store():
+    return ElasticsearchDocumentStore(
+        hosts=host,
+        basic_auth=(username, password),
+        index="permit_condition_embeddings",
+        embedding_similarity_function="cosine",
+        ca_certs=ca_cert if ca_cert else None,
+        verify_certs=True if ca_cert else False,
+    )
+
+template = """
+    You are an expert assistant that helps users find information about permit conditions and reason about them. When answering a question or searching, you can only use the information in the sources provided. \n
+    You must cite all your sources in square brackets with the prefix "doc" for example: Paris is the capital of france [doc:1].\n You must represent the information in the sources accurately and not modify the content at all. \n
+    When you find a matching permit condition, you should enclose it in a Markdown blockquotes (>) to make it clear that it is a direct quote from the source. \n 
+    
+    You will be given a list of permit conditions (sources) with the content of the condition and and ID you can use to cite the source.
```

Comment:
The AI probably won't care, but there's a small typo here with a double 'and' "content of the condition and and ID you can use"